<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="9" failures="7" name="io.cloudboost.CloudGeoPointTest" time="0.142" errors="0" skipped="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/amd64"/>
    <property name="java.vm.version" value="25.60-b23"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=":"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/media/ranjeet/CED23D64D23D52476/cloudboost/JavaSDK/JavaSDK"/>
    <property name="java.runtime.version" value="1.8.0_60-b27"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/endorsed"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="3.13.0-53-generic"/>
    <property name="user.home" value="/root"/>
    <property name="user.timezone" value="Asia/Kolkata"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="user.name" value="root"/>
    <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.x.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="java.home" value="/usr/lib/jvm/java-8-oracle/jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher package"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_60"/>
    <property name="java.ext.dirs" value="/usr/lib/jvm/java-8-oracle/jre/lib/ext:/usr/java/packages/lib/ext"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="sun.boot.class.path" value="/usr/lib/jvm/java-8-oracle/jre/lib/resources.jar:/usr/lib/jvm/java-8-oracle/jre/lib/rt.jar:/usr/lib/jvm/java-8-oracle/jre/lib/sunrsasign.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jsse.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jce.jar:/usr/lib/jvm/java-8-oracle/jre/lib/charsets.jar:/usr/lib/jvm/java-8-oracle/jre/lib/jfr.jar:/usr/lib/jvm/java-8-oracle/jre/classes"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="/usr/share/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="getDataNearFunction" time="0.019">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$2.done(CloudGeoPointTest.java:43)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudGeoPointTest.getDataNearFunction(CloudGeoPointTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
http://localhost:4730/data/sample123/Custom5/find
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="getListPolygonGeowithin" time="0.012">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$3.done(CloudGeoPointTest.java:66)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudGeoPointTest.getListPolygonGeowithin(CloudGeoPointTest.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
{}
http://localhost:4730/data/sample123/Custom5/find
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="updateGeoPoint" time="0.017">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$7.done(CloudGeoPointTest.java:151)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudGeoPointTest.updateGeoPoint(CloudGeoPointTest.java:147)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
{&quot;document&quot;:{&quot;expires&quot;:null,&quot;_isModified&quot;:true,&quot;_type&quot;:&quot;custom&quot;,&quot;ACL&quot;:{&quot;read&quot;:{&quot;allow&quot;:{&quot;role&quot;:[],&quot;user&quot;:[&quot;all&quot;]},&quot;deny&quot;:{&quot;role&quot;:[],&quot;user&quot;:[]}},&quot;write&quot;:{&quot;allow&quot;:{&quot;role&quot;:[],&quot;user&quot;:[&quot;all&quot;]},&quot;deny&quot;:{&quot;role&quot;:[],&quot;user&quot;:[]}}},&quot;location&quot;:{&quot;_isModified&quot;:true,&quot;latitude&quot;:17.9,&quot;_type&quot;:&quot;point&quot;,&quot;coordinates&quot;:[17.9,79.6],&quot;longitude&quot;:79.6},&quot;_tableName&quot;:&quot;Custom5&quot;,&quot;_modifiedColumns&quot;:[&quot;createdAt&quot;,&quot;updatedAt&quot;,&quot;ACL&quot;,&quot;expires&quot;,&quot;location&quot;]},&quot;key&quot;:&quot;9SPxp6D3OPWvxj0asw5ryA==&quot;}
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="getListPolygonGeowithinEqualToLimit" time="0.012">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$4.done(CloudGeoPointTest.java:90)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudGeoPointTest.getListPolygonGeowithinEqualToLimit(CloudGeoPointTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
{}
http://localhost:4730/data/sample123/Custom5/find
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="getListCircleGeowithin" time="0.009">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$5.done(CloudGeoPointTest.java:110)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudGeoPointTest.getListCircleGeowithin(CloudGeoPointTest.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
http://localhost:4730/data/sample123/Custom5/find
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="getListCircleGeowithinEqualToLimit" time="0.026">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$6.done(CloudGeoPointTest.java:131)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudGeoPointTest.getListCircleGeowithinEqualToLimit(CloudGeoPointTest.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
http://localhost:4730/data/sample123/Custom5/find
</system-out>
  </testcase>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="takeLatitudeInRange" time="0.011"/>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="takeLongitudeInRange" time="0.006"/>
  <testcase classname="io.cloudboost.CloudGeoPointTest" name="saveLatitudeLongitude" time="0.016">
    <failure message="java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudGeoPointTest$1.done(CloudGeoPointTest.java:27)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudGeoPointTest.saveLatitudeLongitude(CloudGeoPointTest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
    <system-out>SQL Lite is Up
Driver name: SQLiteJDBC
Driver version: native
Product name: SQLite
Product version: 3.8.11
Table Created
Socket Initializaed
{&quot;document&quot;:{&quot;expires&quot;:null,&quot;_isModified&quot;:true,&quot;_type&quot;:&quot;custom&quot;,&quot;ACL&quot;:{&quot;read&quot;:{&quot;allow&quot;:{&quot;role&quot;:[],&quot;user&quot;:[&quot;all&quot;]},&quot;deny&quot;:{&quot;role&quot;:[],&quot;user&quot;:[]}},&quot;write&quot;:{&quot;allow&quot;:{&quot;role&quot;:[],&quot;user&quot;:[&quot;all&quot;]},&quot;deny&quot;:{&quot;role&quot;:[],&quot;user&quot;:[]}}},&quot;location&quot;:{&quot;_isModified&quot;:true,&quot;latitude&quot;:17.7,&quot;_type&quot;:&quot;point&quot;,&quot;coordinates&quot;:[17.7,78.9],&quot;longitude&quot;:78.9},&quot;_tableName&quot;:&quot;Custom5&quot;,&quot;_modifiedColumns&quot;:[&quot;createdAt&quot;,&quot;updatedAt&quot;,&quot;ACL&quot;,&quot;expires&quot;,&quot;location&quot;]},&quot;key&quot;:&quot;9SPxp6D3OPWvxj0asw5ryA==&quot;}
</system-out>
  </testcase>
</testsuite>